sample/src/App.tsx

```diff
@@ -51,6 +52,15 @@ Sentry.init({
         return context;
       },
     }),
+    new Integrations.HttpClient({
+      // This array can contain tuples of `[begin, end]` (both inclusive),
+      // Single status codes, or a combinations of both.
+      // default: [[500, 599]]
+      failedRequestStatusCodes: [[400, 405], [500, 505], 507],
+      // This array can contain Regexes or strings, or combinations of both.
+      // default: [/.*/]
+      failedRequestTargets: ['http://localhost'],
```

Comment:
Should we just leave the default? otherwise, no errors will be captured if not localhost